### PR TITLE
Encrypt ebs volume for aws

### PR DESCRIPTION
**Pull request by tei-k**
# Description

Encrypt ebs volume

# Done
- cassandra data volume
- cassandra commit log volume
- monitor log volume

### CODE REVIEW

File: modules/aws/cassandra/cassandra.tf [hunk 1]
```diff
@@ -41,12 +41,16 @@ module "cassandra_cluster" {
...
 
+data "aws_ebs_default_kms_key" "current" {}
+
 resource "aws_ebs_volume" "cassandra_data_volume" {
   count = local.cassandra.enable_data_volume && ! local.cassandra.data_use_local_volume ? local.cassandra.resource_count : 0
 
   availability_zone = local.locations[count.index % length(local.locations)]
   size              = local.cassandra.data_remote_volume_size
   type              = local.cassandra.data_remote_volume_type
+  encrypted         = true
```
Comment (feeblefakie) [inline modules/aws/cassandra/cassandra.tf:52]: Sorry to say this now. 
The default should be `encrypted =true` but can you make it configurable ?

Comment (tei-k) [reply modules/aws/cassandra/cassandra.tf]: >The default should be encrypted =true but can you make it configurable ?

OK! Fixed in 1abcd97. (Also deleted `kms_key_id` to use default kms_key - Just my check miss)

File: modules/aws/cassandra/locals.tf [hunk 2]
```diff
@@ -22,6 +22,7 @@ locals {
     resource_type                = "t3.large"
     resource_count               = 3
     resource_root_volume_size    = 64
+    encrypt_ebs_volume           = true
```
Comment (feeblefakie) [inline modules/aws/cassandra/locals.tf:25]:
```suggestion
    encrypt_volume           = true
```
so that we can use the same name for Azure.

Comment (tei-k) [reply modules/aws/cassandra/locals.tf]: Fixed in cf2c465

**Pull request merged**